packages/footer/components/Partners.tsx

```diff
@@ -7,12 +7,11 @@ export const FooterPartners = forwardRef<BoxProps, 'div'>(
       return (
          <Flex
             ref={ref}
-            mt="7"
+            mt="8"
             alignItems="center"
             justifyContent="center"
             flexWrap="wrap"
-            paddingLeft="4"
-            paddingRight="4"
+            gap="3"
```

Comment:
I don't think we want to be using gap any more: https://github.com/iFixit/ifixit/pull/46613

Reply:
See: https://ifixit.slack.com/archives/C01GVQAAGDU/p1678818687071049

@jarstelfox mentioned we want to be using chakra `spacing` prop instead: https://chakra-ui.com/docs/components/stack/usage#usage